src/components/data_submission/NIHDataManagement.js

```diff
@@ -0,0 +1,221 @@
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other informed consent limitations or concerns',
+        }),
+        h(FormField, {
+          id: 'otherReasonForRequest',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherReasonForRequest),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other'
+        }),
+      ]),
+      h(FormField, {
+        isRendered: true, // todo: otherReasonForRequest,
+        id: 'alternativeDataSharingPlanExplanation',
+        defaultValue: initialFormData?.alternativeDataSharingPlanExplanation,
+        onChange,
+        title: 'Explanation for request',
+        validators: [FormValidators.REQUIRED]
+      }),
+      h(FormField, {
+        type: FormFieldTypes.FILE,
+        title: 'Upload your alternative sharing plan.',
+        id: 'alternativeDataSharingPlanFile',
+        onChange,
+      }),
+
+      h(FormField, {
+        type: FormFieldTypes.RADIOGROUP,
+        title: 'Data will be submitted:',
+        id: 'alternativeDataSharingPlanDataSubmitted',
+        defaultValue: initialFormData?.alternativeDataSharingPlanDataSubmitted,
+        options: [
+          {
+            text: 'Within 3 months of the last data generated or last clinical visit',
+            name: 'Within 3 months of the last data generated or last clinical visit',
+            id: 'withinThreeMonths',
+          },
+          {
+            text: 'By batches over Study Timeline (e.g. based on clinical trial enrollment benchmarks)',
+            name: 'By batches over Study Timeline (e.g. based on clinical trial enrollment benchmarks)',
+            id: 'batches',
+          }
+        ],
+        onChange,
+      }),
+      h(FormField, {
+        type: FormFieldTypes.YESNORADIOGROUP,
+        id: 'alternativeDataSharingPlanDataReleased',
+        defaultValue: initialFormData?.alternativeDataSharingPlanDataReleased,
+        title: 'Data to be released will meet the timeframes specified in the NHGRI Guidance for Data Submission and Data Release',
+        onChange,
+      }),
+
+      div({
+        style: {
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+        },
+      }, [
+
+        h(FormField, {
+          id: 'alternativeDataSharingPlanTargetDeliveryDate',
+          style: {
+            width: '45%',
+          },
+          defaultValue: initialFormData?.alternativeDataSharingPlanTargetDeliveryDate,
+          title: 'Target Delivery Date',
+          placeholder: 'Please enter date (YYYY-MM-DD)',
+          validators: [FormValidators.DATE],
+          onChange,
+        }),
```

Comment:
For date fields, we should have a date selection widget here. I think this can be filed as a separate ticket for updating `FormField`.

Reply:
filed ticket!